nengo_bones/scripts/format_notebook.py

```diff
@@ -33,11 +33,10 @@ def format_notebook(nb, fname, verbose=False, prettier=None):  # noqa: C901
 
     # --- Remove bad metadata
     # Should pass `nengo/tests/test_examples.py:test_minimal_metadata`
-    if "kernelspec" in nb.metadata:
-        del nb.metadata["kernelspec"]
-
-    if "widgets" in nb.metadata:
-        del nb.metadata["widgets"]
+    badmeta = ("kernelspec", "widgets")
```

Comment:
Super nitpick but I'm going to change this to a list. I think I was convinced of this by a talk from Alex Gaynor where he said that if you could see the length of collection changing, it should be a list, and if not then it should be a tuple. There's also [this post](https://jtauber.com/blog/2006/04/15/python_tuples_are_not_just_constant_lists/).